### PR TITLE
expand fixes

### PR DESCRIPTION
**Pull request by lennyburdette**
---

**Checklist**

Complete the checklist (and note appropriate exceptions) before the PR is marked ready-for-review.

- [ ] Changes are compatible[^1]
- [ ] Documentation[^2] completed
- [ ] Performance impact assessed and acceptable
- Tests added and passing[^3]
    - [ ] Unit Tests
    - [x] Integration Tests
    - [ ] Manual Tests

**Exceptions**

*Note any exceptions here*

**Notes**

[^1]: It may be appropriate to bring upcoming changes to the attention of other (impacted) groups. Please endeavour to do this before seeking PR approval. The mechanism for doing this will vary considerably, so use your judgement as to how and when to do this.
[^2]: Configuration is an important part of many changes. Where applicable please try to document configuration examples.
[^3]: Tick whichever testing boxes are applicable. If you are adding Manual Tests, please document the manual testing (extensively) in the Exceptions.

### CODE REVIEW

File: apollo-federation/src/sources/connect/expand/visitors/input.rs [hunk 1]
```diff
@@ -102,6 +126,8 @@ impl GroupVisitor<InputObjectTypeDefinitionPosition, InputObjectFieldDefinitionP
             "tried to exit a group not yet visited",
         ))?;
 
+        // dbg!(&definition, &r#type);
```
Comment (nicholascioli) [inline apollo-federation/src/sources/connect/expand/visitors/input.rs:129]: Should this be deleted?

**Review by nicholascioli — APPROVED** [review]
This looks great! A small nit on a commented out line, but feel free to ignore!

File: apollo-federation/src/sources/connect/expand/visitors/input.rs (hunk 1 above)
Comment (lennyburdette) [reply apollo-federation/src/sources/connect/expand/visitors/input.rs:129]: yup!

Comment (lennyburdette) [inline apollo-federation/src/sources/connect/expand/visitors/input.rs:129]:
```suggestion
```

File: apollo-federation/src/sources/connect/expand/visitors/input.rs [hunk 2]
```diff
@@ -102,6 +126,7 @@ impl GroupVisitor<InputObjectTypeDefinitionPosition, InputObjectFieldDefinitionP
             "tried to exit a group not yet visited",
         ))?;
 
+
```
Comment (lennyburdette) [inline apollo-federation/src/sources/connect/expand/visitors/input.rs:129]:
```suggestion
```

**Pull request merged**